chainer/links/loss/negative_sampling.py

```diff
@@ -28,12 +29,13 @@ class NegativeSampling(link.Link):
 
     """
 
-    def __init__(self, in_size, counts, sample_size, power=0.75):
+    def __init__(self, in_size, counts, sample_size, power=0.75, dtype=None):
         super(NegativeSampling, self).__init__()
+        dtype = chainer.get_dtype(dtype)
```

Comment:
import `chainer`.

Reply:
Fixed! Missing `import chainer` line looks imported to the master in #5725, ChainerX integration.